### ISSUE TITLE
"Production capacities and costs of hydrogen" chart should show hydrogen output capacity

### ISSUE THREAD

**Issue by mabijkerk** [quintel/etmodel#3618]
The chart "Production capacities and costs of hydrogen" shows, for each installed hydrogen production technology, the production costs and the installed capacity. This chart should therefore give the user insight into the amount of hydrogen that can be produced at a specific moment. Currently however, not the output capacity of hydrogen is shown, but the input capacity of the energy source used to produce the hydrogen.

In the example below this is demonstrated for dedicated offshore wind turbines. The capacity slider is set at 12 GW and this exact amount is displayed in the chart. This capacity is the electricity output capacity of a wind turbine, not the hydrogen output capacity of the electrolysis.
<img width="1098" alt="Screenshot 2021-02-18 at 10 28 03" src="https://user-images.githubusercontent.com/67338510/108336122-ff49d900-71d3-11eb-88b1-ec383ef38d3a.png">

For other hydrogen production technologies such as steam methane reforming and power-to-hydrogen this is also the case.
I propose to check whether all hydrogen production technologies currently show the installed **input capacity** in this chart and to update the chart to only show **hydrogen output capacities**.

Notifying @ChaelKruip and @michieldenhaan.

**Comment by mabijkerk** [comment]
The discussion above also applies to the table "Flexibility options for excess electricity". In the example below the capacity shown for conversion to heat for district heating is the **heat output** capacity, while the capacity shown for conversion to hydrogen is the **electricity output** capacity.

<img width="603" alt="Screenshot 2021-03-01 at 14 35 01" src="https://user-images.githubusercontent.com/67338510/109504088-6a7c9080-7a9b-11eb-9973-dddf25b1a318.png">

I think that this table should only show **electricity input** capacity, because the table is aimed at dealing with **excess** electricity. What do you think @marliekeverweij and @michieldenhaan?

**Comment by marliekeverweij** [comment]
> I think that this table should only show electricity input capacity, because the table is aimed at dealing with excess electricity. What do you think @marliekeverweij and @michieldenhaan?

I agree 👍

**Comment by mabijkerk** [comment]
@kaskranenburgQ this is one to keep in mind for the hydrogen project. Perhaps we can pick it up when we are updating the charts.

**Comment by kndehaan** [comment]
This issue is solved in hydrogen-improvements and can be closed when it's merged to beta, right?

**Comment by mabijkerk** [comment]
Yes definitely. You can link the pull request if you want @kndehaan.
